src/routes/services.ts

```diff
@@ -31,16 +32,19 @@ export const vpn = {
 				captureException(err, 'Error authenticating device for VPN', { req });
 				res.status(500).send(err);
 			}),
-	clientConnect: (req: Request, res: Response) => {
+	clientConnect: (req: Request, res: Response): Promise<void> => {
 		const body = req.body || {};
 		if (!body.common_name) {
-			return res.sendStatus(400);
+			res.sendStatus(400);
+			return null as any;
```

Comment:
Why is this not just `return`?  That would avoid this odd type casting

Reply:
TS complains again :) It treats `return` as equivalent to `return undefined` and complains the signature is incompatible.

Reply:
Well the signature *is* incompatible - it specifies a promise is returned when it's not